internal/cmd/operator_list_custom_resources.go

```diff
@@ -0,0 +1,71 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/operator-framework/kubectl-operator/internal/cmd/internal/log"
+	"github.com/operator-framework/kubectl-operator/internal/pkg/action"
```

Comment:
Nit: can you separate the `github.com/operator-framework/kubectl-operator` imports into a third group?